modules/tinymce/src/themes/silver/main/ts/ui/core/complex/FontSizeBespoke.ts

```diff
@@ -110,6 +121,34 @@ const getSpec = (editor: Editor): SelectSpec => {
 const createFontSizeButton = (editor: Editor, backstage: UiFactoryBackstage): SketchSpec =>
   createSelectButton(editor, backstage, getSpec(editor));
 
+const getNumberInputSpec = (editor: Editor): NumberInputSpec => {
+  const getConfigFromUnit = (unit: string): Config => {
+    const baseConfig = { step: 1 };
+
+    const configs: Record<string, Config> = {
+      em: { step: 0.1 },
+      px: { step: 1 },
+      pt: { step: 1 }
+    };
+
+    return configs[unit] || baseConfig;
```

Comment:
Should we use `??` instead of `||` here? Are you intentionally wanting `0` to be defaulted?

Reply:
changed